### PR TITLE
ON-5/implementation add instance function 

### PR DESCRIPTION
**Pull request by pankogito**
ON-5
implementation and tests for add instance function

### CODE REVIEW

File: ontology_api/model/model_service.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
 from owlready2 import get_ontology
```
Comment (mrwrob) [inline ontology_api/model/model_service.py:1]: Please reformat code (Ctrl+Alt+L in PyCharm)

File: ontology_api/model/model_service.py [hunk 2]
```diff
@@ -75,3 +77,15 @@ def save_model_as_owl(self, model, model_id, path=None):
     def get_owl_from_model(self, model_id, path=None):
         model = self.__find_model_by_id(model_id)
         return self.save_model_as_owl(model, model_id, path)
+
+    def add_instance(self, class_id, model_id, model_in: MinimalInstanceModelIn):
```
Comment (mrwrob) [inline ontology_api/model/model_service.py:81]: Please add description of this function. I don't get what this is supposed to do.... What class the `model` is? Does it have `classes` method? What is the result of this function? Is it setting something, or returning?

File: ontology_api/tests/test_model_router.py [hunk 3]
```diff
@@ -57,3 +59,36 @@ def test_create_model_with_file_with_error(self):
         os.remove("tests/tmp_owl/testfile.txt")
         #os.remove("testfile.txt")
         self.assertEqual(response.status_code, 422)
+
+    def test_add_instance_happy_path(self):
+        model_router = ModelRouter()
+        response = Response()
+        background_tasks = BackgroundTasks()
```
Comment (mrwrob) [inline ontology_api/tests/test_model_router.py:66]: Are we using this variable?

**Review by mrwrob — CHANGES_REQUESTED** [review]
Please don't forget to reformat the code (Ctrl+Alt+L in PyCharn)

**Review by PLangowski — APPROVED** [review]
Looks good

**Review by mrwrob — APPROVED** [review]
LGTM :+1:

**Pull request merged**